FunctionalTableData/CellStyle.swift

```diff
@@ -57,24 +63,32 @@ public struct CellStyle {
 		
 		cell.tintColor = tintColor
 		
+		if #available(iOS 11.0, *) {
+			cell.insetsLayoutMarginsFromSafeArea = false
+			cell.contentView.insetsLayoutMarginsFromSafeArea = false
+		}
+		
 		if let selectionColor = selectionColor {
 			let selectedBackgroundView = UIView()
 			selectedBackgroundView.backgroundColor = selectionColor
 			cell.selectedBackgroundView = selectedBackgroundView
 		} else {
 			cell.selectedBackgroundView = nil
 		}
+		
+		cell.layer.cornerRadius = cornerRadius
+		cell.layer.masksToBounds = true
```

Comment:
I wonder if `masksToBounds` should only be set if `cornerRadius` is > 0. We don't need this otherwise, and theoretically it makes drawing slower; also, consumers of this API may want to display content that extends beyond the bounds of the cell.